File: PayPal/src/main/java/com/braintreepayments/api/PayPalPaymentAuthCallback.java
package com.braintreepayments.api;

/**
 * Callback for receiving result of
 * {@link PayPalClient#createPaymentAuthRequest(android.content.Context, PayPalRequest, PayPalPaymentAuthCallback)}.
 */
public interface PayPalPaymentAuthCallback {

    /**
     * @param paymentAuthRequest a request used to launch the PayPal web authentication flow
     */
    void onPayPalPaymentAuthRequest(PayPalPaymentAuthRequest paymentAuthRequest);
}

File: PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthCallback.kt
package com.braintreepayments.api.paypal

/**
 * Callback for receiving result of [PayPalClient.createPaymentAuthRequest].
 */
fun interface PayPalPaymentAuthCallback {

    /**
     * @param paymentAuthRequest a request used to launch the PayPal web authentication flow
     */
    fun onPayPalPaymentAuthRequest(paymentAuthRequest: PayPalPaymentAuthRequest?)
}
